### PR TITLE
STANFORD-562 | Add option to add a link button to Text Callout component

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Add option to add a link button to the bottom of the Text Callout component

# Review By (Date)
- Soon

# Criticality
- 4

# Review Tasks

## Setup tasks and/or behavior to test

1. See the 4 examples I put at the bottom of my test page
https://sug-web.matrix.squiz.cloud/report/development/yvonnes-test-page
2. Try adding the Text Callout component on your own in Matrix and play around with the options

**Note:** I notice that currently, the horizontal line in the title isn't on its own line on mobile - it should be in the mock. LMK if you'd like me to fix that (not sure if Squiz devs are on it already).

# Associated Issues and/or People
- JIRA ticket(s) - STANFORD-562
- STANFORD-560

### CODE REVIEW

File: components/text-callout/Component.jsx [hunk 1]
```diff
@@ -80,6 +84,15 @@ export default function TextCallout({
             </p>
           </div>
         )}
+        {buttonText && (
+          <LinkButton
+            buttonText={buttonText}
+            internalUrl={internalLinkUrl}
+            externalUrl={externalUrl}
+            isNewWindow={isNewWindow}
+            className="su-rs-mt-1 su-order-3"
```
Comment (yvonnetangsu) [inline components/text-callout/Component.jsx:93]: These additional classes are added to the `LinkButton` component through the use of `cnb` in the component.

File: components/text-callout/manifest.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "text-callout",
-  "version": "0.0.13",
+  "version": "0.0.15",
   "namespace": "stanford-components",
-  "description": "This component is intended for displaying extra text content with a background colour applied",
+  "description": "This component is intended for displaying extra text content with a background color applied",
```
Comment (yvonnetangsu) [inline components/text-callout/manifest.json:6]: Going with the 🇺🇸 spelling 😆 even though I'm 🇨🇦

File: packages/links/LinkButton.jsx [hunk 3]
```diff
@@ -32,7 +35,10 @@ export function LinkButton({
   return internalUrl || externalUrl ? (
     // eslint-disable-next-line react/jsx-no-target-blank
     <a
-      className="su-group su-inline-block su-button hocus:su-underline md:su-px-30 md:su-pt-12 md:su-pb-14 su-text-18 md:su-text-20 su-transition-colors"
+      className={cnb(
+        "su-group su-inline-block su-button hocus:su-underline md:su-px-30 md:su-pt-12 md:su-pb-14 su-text-18 md:su-text-20 su-transition-colors dark:hocus:su-ring-1 dark:hocus:su-ring-white",
```
Comment (yvonnetangsu) [inline packages/links/LinkButton.jsx:39]: @broleomo @iamrentman I used the TW ring utility here. It achieves the same effect as border, but it's inset, so doesn't cause a layout shift when going to no border (regular state) to 1px border on hocus in dark mode. It's a handy utility 😄 
https://tailwindcss.com/docs/ring-width#inset-rings

**Review by broleomo — APPROVED** [review]
feedback sent via slack. Thanks for updating Yvonne!

**Comment by yvonnetangsu** [comment]
> feedback sent via slack. Thanks for updating Yvonne!

Thank you @broleomo! I'll merge this in then update the Button Row component as well in the other PR 😄

**Pull request merged**